docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    plt.plot(T[idx_motifA:idx_motifA+m], color="C1")
</pre><p>Note that <code>stumpy.stump()</code>,  by default, normalize the subsequences prior to the calculation of matrix profile. Hence, plotting the subsequence <code>T[idx_motifA:idx_motifA+m]</code> is not meaningful here.  You should first z-normalize each subseq and then plot them. </p><p>(To z-normalize the subsequence, you may use <code>z_norm</code> function provided in  <code>stumpy/core.py</code> )</p><p> </p><p>OR, </p><p>you can do <code>p = stumpy.stump(T, m, normalize=False)</code> in the first place. Then, you are allowed to plot the raw subsequences as motif pairs without z-normalizing them.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
If `normalize=True` then, yes, I would plot the `z-norm` of the subsequences and then label them as such. Either go all `normalize=True` OR `normalize=False`. Don't mix and match because the inconsistency will lead to confusion. I vote for `normalize=True` as that is the default behavior and  so you should `z-norm` before plotting